### PR TITLE
Pipeline Configs Documentation

### PR DESCRIPTION
**Pull request by melody-shellman**
## Fixes/Addresses/Summary/Motivation:

- RST Documentation associated with PR #89
- Addresses Issue #120

## Changes proposed in this PR:

Update strategic_water_managent\index.rst

- Define different parameters depending on distance based vs capacity based
- Update Pipeline Capacity Construction cost with note about input vs calculated capacities
- Update Construction of Capacity Expansion Cost for 2 different cost formulations depending on distance based vs capacity based

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my
contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.md file
   at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has
   rights to intellectual property that includes these contributions, I represent that I have
   received permission to make contributions and grant the required license on behalf of that
   employer.

### CODE REVIEW

**Review by NienkeWagenaar — APPROVED** [review]
Made a minor change, kappa wasn't shows as math. Other than that it looks good to me!

File: docs/model_library/strategic_water_management/index.rst [hunk 1]
```diff
@@ -664,13 +672,16 @@ Sets the flow capacity in a given pipeline during a given time period. Different
 
     \textcolor{red}{F_{l,l ̂,[t]}^{Capacity}} = \textcolor{green}{σ_{l,l ̂}^{Pipeline}}+∑_{d∈D}\textcolor{green}{δ_{d}^{Pipeline}}⋅(\textcolor{red}{y_{l,l ̂,d}^{Pipeline}}+\textcolor{red}{y_{l ̂,l,d}^{Pipeline}} )+\textcolor{red}{S_{l,l ̂}^{PipelineCapacity}}
 
-.. note:: Parameter :math:`δ_{d}^{Pipeline}` will be calculated as follows:
+.. note::
+
+    δ can be input by user or calculated. If the user chooses to calculate pipeline capacity, the parameter will be calculated by the equation below where :math:`{κ_{l,l}}` is Hazen-Williams constant and ω is Hazen-Williams exponent as per Cafaro & Grossmann (2020) and d represents the pipeline diameter as per the set d∈D.
```
Comment (tarnold17) [inline docs/model_library/strategic_water_management/index.rst:677]: I'd recommend including the full citation to Cafaro and Grossman (2020) at the bottom of the page.

Comment (melody-shellman) [reply docs/model_library/strategic_water_management/index.rst]: implemented!

**Review by tarnold17 — APPROVED** [review]

**Review by NienkeWagenaar — APPROVED** [review]
Nice!

**Pull request merged**